src/app/qgsoptions.cpp

```diff
@@ -2300,3 +2300,25 @@ void QgsOptions::scaleItemChanged( QListWidgetItem *changedScaleItem )
   addScaleToScaleList( changedScaleItem );
   mListGlobalScales->setCurrentItem( changedScaleItem );
 }
+
+double QgsOptions::getZoomFactorValue(QSpinBox *zoomFactorSpinBox)
+{
+  // Get the decimal value for zoom factor. This function is needed because the zoom factor spin box is shown as a percent value.
+  // The minimum zoom factor value is 1.01
+  if ( zoomFactorSpinBox->value() == zoomFactorSpinBox->minimum() ) return 1.01;
```

Comment:
Not sure if that's done by astyl automatically meanwhile. We normally break after every if

    if ( xxx )
      return 1.01;